src/doubleTrouble.js

```diff
@@ -24,7 +24,15 @@
  * @return {number[]} - array of remained numbers
  */
 function doubleTrouble(numbers, target) {
-  // write code here
+  const numbersNew = [...numbers];
```

Comment:
`modifiedNumbers` is a better name

Reply:
The name is changed:
![image](https://user-images.githubusercontent.com/32244670/63334584-6323e500-c344-11e9-86fc-9f342c99be48.png)